src/utils/aeternity/client.ts

```diff
@@ -0,0 +1,78 @@
+// @ts-ignore
+import { Node, Universal } from '@aeternity/aepp-sdk'
+import { reactive, toRefs } from 'vue'
+import { COMPILER_URL, NETWORKS } from './configs'
+// @ts-ignore
+import identity from './contracts/Idenitity.aes'
+
+export const aeClient = reactive<{
+    sdk: any;
```

Comment:
I think most of the boilerplate users won't want to use typescript until it is supported on the SDK side.

Reply:
I see some typescript support already started on the SDK https://github.com/aeternity/aepp-sdk-js/issues?q=typescript

Reply:
Yea, but currently it is only for utility functions (< 10% of code). I assume that boilerplate is for fast prototyping, so dealing with typescript may be an unwanted thing by default. Proposing to drop typescript and if needed user may do [`vue add typescript`](https://cli.vuejs.org/core-plugins/typescript.html#installing-in-an-already-created-project) after cloning the boilerplate.

Reply:
ah okay, removed typescript